monitoring-usage/webhooks.md

```diff
@@ -126,6 +127,16 @@ Additional properties in data for just `transcription.failed`:
 | -------- | ------------------------------------------------------------------------- |
 | error    | The error message that describes why the transcription failed to process. |
 
+## Cloud Recording Data properties
+
+Properties in `data` that are common to all transcription webhook events:
```

Comment:
```suggestion
Properties in `data` that are common to all cloud recording webhook events:
```
Alternatively just say something like this:
```
Properties in `data` for the `recording.finished` webhook event
```
Since we only have one recording event at the moment.